### PR TITLE
[C-1326] Fix open in app drawer to open the app properly

### PR DESCRIPTION
**Pull request by Kyle-Shanks**
### Description
Fixed the Open in app drawer to use deep link to open the mobile app properly

### Dragons

May be some slight bugs using a different link, but appears to be working

### How Has This Been Tested?

Manually tested

### How will this change be monitored?

N/a

### Feature Flags ###

N/A

### CODE REVIEW

File: packages/web/src/components/app-redirect-popover/components/AppRedirectPopover.tsx [hunk 1]
```diff
@@ -119,7 +119,8 @@ const AppRedirectPopover = ({
   const onClick = () => {
     onBeforeClickApp()
     const pathname = getPathname()
-    const newHref = `https://redirect.audius.co${APP_REDIRECT}${pathname}`
+    const newHref = `audius://${pathname}`
```
Comment (raymondjacobson) [inline packages/web/src/components/app-redirect-popover/components/AppRedirectPopover.tsx:122]: nice. this redirect.audius.co thing is super broken.
does this work if the app is not installed?

File: packages/mobile/src/components/navigation-container/NavigationContainer.tsx [hunk 2]
```diff
@@ -124,6 +124,9 @@ const NavigationContainer = (props: NavigationContainerProps) => {
       }
     },
     getStateFromPath: (path, options) => {
+      // Add leading slash if it is missing
+      if (path[0] !== '/') path = `/${path}`
```
Comment (dylanjeffers) [inline packages/mobile/src/components/navigation-container/NavigationContainer.tsx:128]: Catch of the century

**Review by dylanjeffers — APPROVED** [review]
Lgtm, also just checking that the redirect part is needed for this pr or if it's for the wallet connect piece?

File: packages/web/src/components/app-redirect-popover/components/AppRedirectPopover.tsx (hunk 1 above)
Comment (sliptype) [reply packages/web/src/components/app-redirect-popover/components/AppRedirectPopover.tsx:122]: No this doesn't work when the app isn't installed, I assume that's why we had the redirect thing?

**Review by dylanjeffers — APPROVED** [review]
Nice

**Pull request merged**